core/backend/src/BackendHubAccess.ts

```diff
@@ -10,24 +10,11 @@ import { AccessToken, GuidString, Id64String, IModelHubStatus } from "@itwin/cor
 import {
   BriefcaseId, ChangesetFileProps, ChangesetIdWithIndex, ChangesetIndex, ChangesetIndexAndId, ChangesetIndexOrId, ChangesetProps, ChangesetRange,
   IModelError, IModelVersion, LocalDirName, LocalFileName,
+  LockState,
 } from "@itwin/core-common";
 import { CheckpointProps, DownloadRequest, ProgressFunction } from "./CheckpointManager";
 import type { TokenArg } from "./IModelDb";
 
-/** The state of a lock.
- * @public
- */
-export enum LockState {
```

Comment:
`export { LockState } from "@itwin/core-common";` to the core-backend.ts file I assume? Just pushed that

Reply:
In BackendHubAccess, something like:
```
import { LockState as CommonLockState } from "@itwin/core-common";

/** @deprecated in 4.7 Use [LockState]($common)
 * @public
 */
export LockState = CommonLockState;

Reply:
`export type` won't work for `enum` since it's a type and value.
Could try `export { CommonLockState as LockState };`
The point is to deprecate the one in core-backend so we can delete it at some point.

Reply:
the deprecated tag wasn't able to show up with export { commonlockstate as Lockstate} so going with duplicate and deprecate approach.